### ISSUE TITLE
Why use your own backgrounds?

### ISSUE THREAD

**Issue by bongbonglemon** [senguptaumd/Background-Matting#22]
I read the Adobe paper about their dataset. I also dropped them a email. To my understanding they have 455 ground truth foregrounds and alpha mattes that they put onto 100 backgrounds each. I'm still awaiting their reply as to whether their dataset includes those 100 stand-alone backgrounds. Meanwhile I would like to hear from yourself as to why you decided to make your own set of synthetic composite with backgrounds drawn from MS COCO. Thank you.

**Comment by senguptaumd** [comment]
<img width="346" alt="Screen Shot 2020-04-15 at 11 13 42 AM" src="https://user-images.githubusercontent.com/34743495/79354298-38f1e980-7f0a-11ea-8c8d-7027b407c7ef.png">

Please read this section from the Deep Image Matting (Adobe) paper. They randomly pick 100 backgrounds per object from MS-COCO dataset. DIM dataset do not contain any background. It has 450 objects with alpha matte. These objects have nearly uniform backgrounds s.t. a photoshop artist can easily create an alpha matte from it. However, that is not good enough for training as there is not enough variability in the background. Compositing 450 images into 100 backgrounds each with different crop help in creating variation in training data.

Take a look at this amazing training code for DIM to better understand it: https://github.com/foamliu/Deep-Image-Matting-PyTorch
